### DISCUSSION TITLE
Hiding Pubs from community members in the Community Overview

### DISCUSSION THREAD

**Discussion by PatienceKey98** [knowledgefutures/pubpub#2517] [Support]
Hi! I'm currently utilizing PubPub as a platform for civic engagement, however, I want to limit access of certain Pubs to a defined set of members.

Now, this all works fine by managing access through my site's main webpage. However, any community member that signs up to my site can look at all the released Pubs in the Community Overview Tab. 

Now, I can hide collections from public view in this Tab, but not the individual Pubs. How can I restrict the visibility of these released Pubs in the Community Overview Tab? I don't want any participant to be poking around where they shouldn't.

**Comment by gabestein** [comment]
Hello! Currently, there's no way to hide Pubs from someone who is a member at the community level, although we're thinking of ways we could enable that.

For now, the best way to restrict access to Pubs is to add Members to Collections you want them to be able to see/manage, rather than the Community itself. This will effectively restrict their access only to Pubs within those Collections.
